custom_components/opnsense/switch.py

```
@@ -384,87 +454,65 @@
+        self._available = True
+        self._attr_extra_state_attributes = {}
+        for attr in ["id", "name"]:
+            self._attr_extra_state_attributes[f"service_{attr}"] = self._service.get(
+                attr, None
+            )
+        self.async_write_ha_state()
+        _LOGGER.debug(
+            f"[OPNsenseServiceSwitch handle_coordinator_update] Name: {self.name}, available: {self.available}, is_on: {self.is_on}, extra_state_attributes: {self.extra_state_attributes}"
+        )
 
     async def async_turn_on(self, **kwargs) -> None:
```

Ruff

Missing type annotation for `self` in method

(ANN101)